lib/elixir/test/elixir/code_test.exs

```diff
@@ -100,6 +100,19 @@ defmodule CodeTest do
     assert Code.string_to_quoted("a.1") == {:error, {1, "syntax error before: ", "1"}}
   end
 
+  test "string_to_quoted/1 for presence of sigils terminators" do
+    assert Code.string_to_quoted("~r/foo/") == {:ok, {:sigil_r, [terminator: '/', line: 1],
```

Comment:
Can you please break it to the next line? Something like:

    assert Code.string_to_quoted("~r/foo/") ==
           {:ok, {:sigil_r, [terminator: '/', line: 1],...}